scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -161,8 +161,14 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                                 <span translate>Max. number of items</span>
                             </div>
                             <div class="form__row-item form__row-item--no-grow">
-                                <input id="max_items" type="number" min="0" max="99" class="input-small line-input"
-                                                                                     ng-model="vocabulary.field_options.multiple_items.max_items" style="width: 60px; text-align: center;">
+                                <input id="max_items" name="max_items" type="number"
+                                        min="2" max="99" class="input-small line-input"
```

Comment:
you should change it here too `min="2"`

Reply:
I had a discussion with Karel, and he said that we are talking about multiple uploads, so it must start with 2, not 1.